mobilizon_reshare/publishers/platforms/utils.py

```diff
@@ -0,0 +1,16 @@
+def fqdn_to_uri(fqdn: str):
```

Comment:
we are not doing this for any other publisher and it would introduce an unexpected behavior. If the user puts a bad url in the config, we should just fail instead of introducing obscure behaviors behind their back